.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
```

Comment:
Should we reuse the maximum in `tox` as well ? for example, `tox -e bandit` is running `bandit -b bandit.json -r oauthlib` and this github step runs `bandit --recursive --skip B101 . || true`. 
Would imagine better for developers to run those checks from their machines directly instead of from github actions ? thoughts?

Reply:
pre-commit is far cooler than tox and it can be run locally and in CI.